config/vaccines.yml

```diff
@@ -6,23 +6,33 @@
     - system: 'http://hl7.org/fhir/sid/cvx'
       code: '213'
   vaccines:
-    - product_name: 'Moderna COVID-19 Vaccine'
-      num_doses: 2
+    - product_name: 'Moderna COVID-19 Vaccine (Non-US tradename: Spikevax)'
+      num_doses: 3
       product_codes:
         - system: 'http://hl7.org/fhir/sid/cvx'
           code: '207'
-    - product_name: 'Pfizer-BioNTech COVID-19 Vaccine'
-      num_doses: 2
+    - product_name: 'Pfizer-BioNTech COVID-19 Vaccine (Non-US tradename: COMIRNATY)'
+      num_doses: 3
       product_codes:
         - system: 'http://hl7.org/fhir/sid/cvx'
           code: '208'
     - product_name: 'Janssen (J&J) COVID-19 Vaccine'
-      num_doses: 1
+      num_doses: 3
```

Comment:
Are we sure J&J should go from 1 to 3? Might be simplest to just allow 3, but I'm not sure we would expect anyone to be in a position where they had 3 doses of J&J. Ticket is somewhat unclear on this, thoughts @hackrm?

Reply:
J&J should only allow for one dose, so this change can be reverted. But I still understand that throguh the UI the user could theoretically be allowed to create a 3rd J&J dose (currently I know the user can configure a 2nd J&J dose).

Reply:
Sorry for the quick reversal, but it appears that J&J will be requesting approval for booster shots. So I'd recommend setting this to 2 actually.

Reply:
531ef49b